util/util.go

```diff
@@ -57,3 +58,19 @@ func WriteTeriminateMessage(message string) {
 		panic(err)
 	}
 }
+
+// Merge the two parameters Slice and overwriting duplicate keys,
```

Comment:
Note: this function doesn't actually overwrite duplicate keys... it just takes the first key. Is this what we want?

Reply:
yes.  passing map in order, the first one take priority

Reply:
Got it. Can we update this comment to reflect that there is no overwriting being done?